### DISCUSSION TITLE
[Feature] Granular control over what shows up on main timeline, storys and search results

### DISCUSSION THREAD

**Discussion by chriexpe** [immich-app/immich#10422] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'm well aware about archiving photos, but good luck doing that when you have, like in my case 55k of photos and 16k of videos, so ideally what I think could be done is:

- Setting that hide content based on tags generated by CLIP (or just straight up NSFW content, bank or receipt prints, person...).

- This one could be a simple fix, but Immich doesn't have such functionality yet where it keeps the photos on same album as they where originally on the device (like Camera, Downloads, Whatsapp, Telegram), so for example you could simply toggle it to show only images that you actually took.

- Have one checkbox near search bar (or inside it, alongside options) that, when checked, it'll search everything, including those contents that were originally hidden - Yes, I'm aware about advanced search, but sometimes, or, most of the times you just want to search without hassling with any setting, yet you want to be safe in not having results that you don't want.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2451 and several other requests.
